app/validation.py

```diff
@@ -114,14 +114,19 @@ def validate_updater_json_or_400(submitted_json):
     try:
         get_validator('services-update').validate(submitted_json)
     except ValidationError as e1:
-        abort(400, "JSON validation error: {}".format(e1.message))
+        try:
+            msg = e1.message
+        except AttributeError:
+            msg = str(e1)
+
+        abort(400, "JSON validation error: {}".format(msg))
```

Comment:
should be only abort(400, "JSON Validation error: {}".format(str(e1))

Reply:
This looks fine to me for now.  We can think about changing it in the future.